probatus/interpret/model_interpret.py

```diff
@@ -177,17 +180,23 @@ def _prep_shap_related_variables(clf, X, y, approximate=False, verbose=0, **shap
             expected_value = expected_value[1]
 
         # Initialize tree dependence plotter
-        tdp = TreeDependencePlotter(clf, verbose=verbose).fit(X, y, precalc_shap=shap_values)
+        tdp = TreeDependencePlotter(clf, verbose=verbose).fit(X, y, column_names=column_names, class_names=class_names,
+                                                              precalc_shap=shap_values)
         return shap_values, expected_value, tdp
 
 
-    def compute(self):
+    def compute(self, return_scores=False):
         """
         Computes the DataFrame, that presents the importance of each feature.
 
+        Args:
+            return_scores (bool, optional):
+                Flag indicating whether the method should return a tuple (feature importances, train score,
+                test score), or feature importances. By default the second option is selected.
```

Comment:
I had to remove it in the end, because the signature of compute in this class would be different than the signature in the abstract method from `BaseFitComputePlotClass`. This causes that the pipeline crashes. In the future we should figure out how to add type hinting across probatus consistently